sim/common/dram_sim.cpp

```diff
@@ -41,11 +41,11 @@ class DramSim::Impl {
 		dram_config["MemorySystem"]["DRAM"]["impl"] = "HBM2";
 		dram_config["MemorySystem"]["DRAM"]["org"]["preset"] = "HBM2_8Gb";
 		dram_config["MemorySystem"]["DRAM"]["org"]["density"] = 8192;
+		dram_config["MemorySystem"]["DRAM"]["org"]["channel"] = 8;
```

Comment:
In Ramulator, this seems to create a new Node class per channel that can each independently handle issues. I set this value to 8 because HBM2 have up to 8 channels. I'm still trying to figure out how this is working in Ramulator as I believe this relates to my problem with source_id in requests.